app/components/provider_interface/provider_permissions_list_component.html.erb

```diff
@@ -1,14 +1,18 @@
 <% visible_provider_permissions.each do |provider_permission| %>
```

Comment:
I just noticed this hides the provider altogether when no permissions are set:

When manage users permission is set for Oriel:
![Screenshot 2020-05-18 at 09 24 44](https://user-images.githubusercontent.com/642279/82190797-7e2e8000-98e9-11ea-98c4-a362551830d8.png)

When no permissions are set:
![Screenshot 2020-05-18 at 09 25 53](https://user-images.githubusercontent.com/642279/82190869-96060400-98e9-11ea-984e-755bf3a8decd.png)

I'll add a separate card

Reply:
That's odd I definitely see the No permissions message:
![image](https://user-images.githubusercontent.com/93511/82209705-a035fb80-9905-11ea-9bbd-84505a2b785d.png)

Reply:
Probably me mixing up my branches locally, I'll check

Reply:
My bad: I removed `manage users` permission for my current user on one of the provider I had access to. Not a bug